tests/suites/test_suite_psa_crypto.function

```diff
@@ -6813,6 +7294,481 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_ECP_RESTARTABLE */
+void verify_hash_fail_interruptible( int key_type_arg, data_t *key_data,
+                                     int alg_arg, data_t *hash_data,
+                                     data_t *signature_data,
+                                     int expected_start_status_arg,
+                                     int expected_complete_status_arg,
+                                     int max_ops_arg )
+{
+    mbedtls_svc_key_id_t key = MBEDTLS_SVC_KEY_ID_INIT;
+    psa_key_type_t key_type = key_type_arg;
+    psa_algorithm_t alg = alg_arg;
+    psa_status_t actual_status;
+    psa_status_t expected_start_status = expected_start_status_arg;
+    psa_status_t expected_complete_status = expected_complete_status_arg;
+    psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
+    uint32_t num_ops = 0;
+    uint32_t max_ops = max_ops_arg;
+    size_t num_ops_prior = 0;
+    size_t num_completes = 0;
+    size_t min_completes = 0;
+    size_t max_completes = 0;
+    psa_verify_hash_interruptible_operation_t operation =
+                              psa_verify_hash_interruptible_operation_init( );
+
+    PSA_ASSERT( psa_crypto_init( ) );
+
+    psa_set_key_usage_flags( &attributes, PSA_KEY_USAGE_VERIFY_HASH );
+    psa_set_key_algorithm( &attributes, alg );
+    psa_set_key_type( &attributes, key_type );
+
+    PSA_ASSERT( psa_import_key( &attributes, key_data->x, key_data->len,
+                                &key ) );
+
+    psa_interruptible_set_max_ops( max_ops );
+
+    interruptible_signverify_get_minmax_completes(max_ops,
+            expected_complete_status, &min_completes, &max_completes);
+
+    num_ops_prior = psa_verify_hash_get_num_ops( &operation );
+    TEST_ASSERT( num_ops_prior == 0 );
+
+    /* Start verification. */
+    actual_status = psa_verify_hash_start( &operation, key, alg,
+                                           hash_data->x, hash_data->len,
+                                           signature_data->x,
+                                           signature_data->len );
+
+    TEST_EQUAL( actual_status, expected_start_status );
+
+    if( expected_start_status != PSA_SUCCESS )
+    {
+        actual_status = psa_verify_hash_start(&operation, key, alg,
```

Comment:
[non-blocker] As Gilles said for sign, I think this belongs in the `state_test` function and here it's more useful to test the case of "start failed but user forgot to check and immediately called complete".

Reply:
#7112